proposed/0002-release-schedule.md

```diff
@@ -0,0 +1,87 @@
+# Release Schedule
+
+## Summary
+
+Defining the Schedule and rules for new releases
+
+## Motivation
+
+Its important to have a reliability for long term plans, we can offer this by following a consistent schedule with hard rules.
+Different People/Users have different needs. This RFC allows to find a middleground, which satisfies all of our users, but also will show us, what our users actually need and want, providing a base for a decision which will be seen as good enough to not need a bigger change for at least a few years.
+With this RFC we then also have something to reference in case of future arguments about Releases and what gets into them.
+
+## Detailed Explanation
+
+With this RFC we will define a basic scheme with rules by which we will do releases, have and create release branches, and also by which we decide what kind of changes get when merged into which release branch.
+
+## Rationale and Alternatives
+
+
+{{Discuss 2-3 different alternative solutions that were considered. This is required, even if it seems like a stretch. Then explain why this is the best choice out of available ones.}}
+
+### Alternative 1
+
+- All new PRs are based on "main" branch (resolves confusion about where to submit PRs)
+- Release timing is flexible (determined by volunteer time availability/motivation) but ideally no more than 1 MAJOR release per year
+- Feature releases must be either MINOR or MAJOR as determined by PRs being merged (tag PRs that should result in major release with "MAJOR")
+- Only significant security patches or regression fixes can be released immediately in a PATCH release
+- PATCH fixes will *only* be back-ported to MAJOR versions that are less than 2 years (730 days) old
+- Old major releases get their own branch only for the purpose of back-porting PATCH fixes (v19, v20, v21, etc)  
```

Comment:
I think these cross with https://github.com/OpenMage/rfcs/pull/6, it could be ok (I like this alternative the way it's written now) but these "where do patches go and for how long" sections are missing from alternative 3, which kinda doesn't make sense anymore.

Reply:
Ahh, thanks, I forgot about that PR, these are indeed very similar.. I think branching and release strategy are so closely related they should both be resolved together, no?

Reply:
Yes, Alternative 3 was missing a lot of detail and is not necessarily different from Alternative 1 except the more rigid timing of releases so I removed it.
@Flyingmana or anyone else feel free to add it back, but with more detail about maintenance of old versions. :)

Reply:
I think (actually, if I'm not mistaken, that's also @Flyingmana opinion) that this PR should be only about release schedule (which also defines how many branches we are maintaining) and then the other one would be different and still needed.

but the way alternative 1 is phrased now, makes alternative 3 useless

Reply:
also, this line

`- Old major releases get their own branch only for the purpose of back-porting PATCH fixes (v19, v20, v21, etc)  `

should have a precise number of branches that patches are ported, otherwise it's again open to intepretation, which is what we're trying to avoid.

Reply:
these modifications are not big but change the RFC quite a bit and I think they should have been done in the drafting phase, we should not vote now until we get everything fixed for everybody.